docs/dietpi_tools.md

```diff
@@ -426,11 +426,63 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
 
 === "DietPi survey"
 
-    This option will upload information on a anonymous base. The aggregated results of all uploads can be seen on the [DietPi survey statistics website](https://dietpi.com/survey/).  
-    Run `dietpi-survey`.
+    #### What is DietPi Survey?
+
+    DietPi Survey allows the DietPi project to obtain general information regarding your system and installed software.
 
     ![DietPi-Survey screenshot](assets/images/dietpi-survey.jpg)
 
+    - No private data is sent. No one can identify you. No IP address is obtained.
+    - This data allows the DietPi project team to focus and improve areas based on popularity. Ensuring the most common devices and software titles will receive the most support and improvements.
+    - By opting out, you are potentially preventing DietPi from achieving the best future possible experience for everyone.
+
+    #### How often does DietPi Survey data get sent?
+
+    The DietPi Survey data file is sent when you install software with `dietpi-software` and update DietPi.
+
+    #### How big is the data upload?
+    The data we upload is tiny (1 KB) and won't effect your internet bandwidth or system performance.
+
+    #### What Data is sent?
+    In the command `dietpi-survey` the exact copy of the data we receive can be viewed. The file we receive is written in bash code to allow us faster report page creation for <https://dietpi.com/survey>.
+
+    Following is an example how this file may look like. It is from a very basic system, so only two software packages (#103, #104) are installed.
```

Comment:
It's Dropbear and DietPi-RAMlog btw. Shall we add the info that `dietpi-software list | grep ' =2'` can be used to get the IDs, names and info of software installed via `dietpi-software`?

Reply:
Ok, added this info.